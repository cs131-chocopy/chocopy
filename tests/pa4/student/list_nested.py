a:[[int]]=None
b:[[str]]=None
a=[[1,2,3],[4,5,6]]
print(a[1][1])
a[1][1]=114
print(a[1][1])
a[1]=[1,514,2]
print(a[1][1])
b=[["1","2","3"],["4","5","6"]]
print(b[0][2])
b[0][2]="1919810"
print(b[0][2])
b[0]=["AAA","BBB","CCC"]
print(b[0][2])