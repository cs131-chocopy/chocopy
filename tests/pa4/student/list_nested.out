5
114
514
3
1919810
CCC
